### ISSUE TITLE
Dual Sim: Sometimes a running call does not show the hangup button

### ISSUE THREAD

**Issue by delijati** [ubports/ubuntu-touch#1703]
- Device: BQ Aquaris 4.5
- Channel: Stable
- Build: OTA-16

### Steps to reproduce
Dual Sim:
Sometimes a running call does not show the hangup button it stays green oO

**Comment by lduboeuf** [comment]
confirmed for me as well, but randomly ( not often )

**Comment by UniversalSuperBox** [comment]
@lduboeuf, does this issue only occur on the E4.5?

**Comment by lduboeuf** [comment]
@UniversalSuperBox idk :-|

**Comment by delijati** [comment]
@lduboeuf are you also using two sim's?

**Comment by lduboeuf** [comment]
@delijati no, but i sometimes have that kind of issue, but it becomes rare now. Doesn't have it since 1 month
